src/wazuh_modules/vulnerability_detector/wm_vuln_detector_db.h

```diff
@@ -193,7 +193,7 @@ static const char *vu_queries[] = {
     [VU_AGENT_PACKAGE_VERSION] = "SELECT SOURCE, PACKAGE_NAME, VERSION, SRC_VERSION, ARCH, VENDOR, REFERENCE, TYPE FROM AGENTS WHERE AGENT_ID = ?;",
     [VU_UPDATE_CVE] = "UPDATE " CVE_TABLE " SET PACKAGE = ?, OPERATION = ? WHERE OPERATION = ?;",
     [VU_UPDATE_CVE_NOT_FIXED] = "UPDATE " CVE_TABLE " SET PACKAGE = ?, OPERATION = ?, OPERATION_VALUE = ? WHERE OPERATION = ?;",
-    [VU_UPDATE_CVE_VAL] = "UPDATE " CVE_TABLE " SET OPERATION = ?, OPERATION_VALUE = ?, ARCH_ID = ? WHERE OPERATION = ?;",
+    [VU_UPDATE_CVE_VAL] = "UPDATE " CVE_TABLE " SET OPERATION = ?, OPERATION_VALUE = ?, ARCH_ID = (CASE WHEN ARCH_ID != 0 THEN ARCH_ID ELSE ? END) WHERE OPERATION = ?;",
```

Comment:
Now this query has a conditional, but it isn't considered here
https://github.com/wazuh/wazuh/blob/8b5e8e91a4b3574b096bc5f4a1e9a2e25143216d/src/wazuh_modules/vulnerability_detector/wm_vuln_detector.c#L3573

We could have more bindings than parameters.

Reply:
After some testing, I can discard this message.